crates/compiler/test_mono/generated/when_nested_maybe.txt

```diff
@@ -1,6 +1,6 @@
 procedure Num.19 (#Attr.2, #Attr.3):
-    let Num.291 : I64 = lowlevel NumAdd #Attr.2 #Attr.3;
-    ret Num.291;
+    let Num.306 : I64 = lowlevel NumAdd #Attr.2 #Attr.3;
+    ret Num.306;
```

Comment:
@folkertdev I wonder if there's a way we can figure out to avoid these diffs everytime a builtin changes. I don't have any good ideas, but maybe something to keep in the back of our minds.

Reply:
richard's ideas for making module depend solely on their contents for caching reasons should do that I think? or at least we can try to make that work in that context.